cdap-common/src/main/java/org/apache/twill/filesystem/FileContextLocation.java

```diff
@@ -80,6 +80,32 @@ public boolean createNew() throws IOException {
   }
 
   @Override
+  public boolean createNew(String permission) throws IOException {
```

Comment:
@gokulavasan Maybe other classes that are copied over from Twill can also be removed then. Can you check?